src/presentation/http/router/note.test.ts

```diff
@@ -49,4 +52,67 @@ describe('Note API', () => {
       expect(body).toStrictEqual({ message: 'Note not found' });
     });
   });
-});
+
+  describe('GET note/:notePublicId ', () => {
+    test('Returns note by public id with 200 status', async () => {
+      const expectedStatus = 200;
+
+      const expectedNote = {
+        'id': 2,
+        'publicId': 'Pq1T9vc23Q',
```

Comment:
API should not return internal id and "publicId". It should return only "id" which is public id. But it seems that it is not implemented yet. I'd suggest leaving a `@todo` for that.